nemo/collections/nlp/data/text_normalization/constants.py

```diff
@@ -0,0 +1,57 @@
+DECODE_CTX_SIZE = 3
+LABEL_PAD_TOKEN_ID = -100
+
+# Task Prefixes
+ITN_PREFIX = str(0)
+TN_PREFIX = str(1)
+
+# Tagger Labels Prefixes
+B_PREFIX = 'B-' # Denote beginning
+I_PREFIX = 'I-' # Denote middle
+TAGGER_LABELS_PREFIXES = [B_PREFIX, I_PREFIX]
+
+# Modes
+TN_MODE = 'tn'
+ITN_MODE = 'itn'
+JOINT_MODE = 'joint'
+MODES = [TN_MODE, ITN_MODE, JOINT_MODE]
+
+# Instance Directions
+INST_BACKWARD = 'BACKWARD'
+INST_FORWARD  = 'FORWARD'
+INST_DIRECTIONS = [INST_BACKWARD, INST_FORWARD]
+
+# TAGS
+TASK_TAG = 'TASK'
+SAME_TAG = 'SAME'
+PUNCT_TAG = 'PUNCT'
+TRANSFORM_TAG = 'TRANSFORM'
```

Comment:
difference between task and transform?

Reply:
Added comments to explain different tags in the code.